seismostats/plots/statistical.py

```diff
@@ -5,91 +5,59 @@
 from scipy.stats import norm
 
 # Own functions
-from seismostats.analysis.bvalue import BPositiveBValueEstimator, estimate_b
+from seismostats.analysis.bvalue.base import BValueEstimator
+from seismostats.analysis.bvalue.classic import ClassicBValueEstimator
 
 
 def plot_mc_vs_b(
     magnitudes: np.ndarray,
     mcs: np.ndarray,
-    delta_m: float = 0.1,
-    method: str = "classic",
+    delta_m: float,
+    b_method: BValueEstimator = ClassicBValueEstimator,
     confidence_intvl: float = 0.95,
     ax: plt.Axes | None = None,
     color: str = "blue",
+    legend: str = None,
```

Comment:
this is 
a) not mentioned in the docstring and
b) not a legend but a label - I would even call it plot_label or call it label and clearly mention in the docstring that it is the plot label.